### PR TITLE
[2.13-maintenance] revert addition of property tests

### PR DESCRIPTION
**Pull request by roberth**
# Motivation

Fix the release branch, and do not introduce dependency in patch release, as requested.

# Context

 - https://github.com/NixOS/nix/pull/7664#issuecomment-1402260180

Maintainers: tick if completed or explain if not relevant

 - [ ] agreed on idea
 - [ ] agreed on implementation strategy
 - [ ] tests, as appropriate
   - functional tests - `tests/**.sh`
   - unit tests - `src/*/tests`
   - integration tests
 - [ ] documentation in the manual
 - [ ] code and comments are self-explanatory
 - [ ] commit message explains why the change was made
 - [ ] new feature or bug fix: updated release notes

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -82,9 +82,7 @@
         });
 
         configureFlags =
-          [
-            "CXXFLAGS=-I${lib.getDev rapidcheck}/extras/gtest/include"
```
Comment (Ericson2314) [inline flake.nix:86]: There is more `CXXFLAGS` in this file thanks to a backported fix.

**Comment by roberth** [comment]
My god, this should be easy and barely depend on human review at all, but here I am fixing this again 6 hours after opening this.

File: flake.nix (hunk 1 above)
Comment (roberth) [reply flake.nix:86]: Done. No more `CXXFLAGS` in any `.nix` now.

File: src/libstore/tests/libstoretests.hh [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-    protected:
-        LibStoreTest()
-            : store(openStore("dummy://"))
-        { }
-
-        ref<Store> store;
-};
-
-
-} /* namespace nix */
```
[the remark this answers is not in the document]
Comment (roberth) [reply src/libstore/tests/libstoretests.hh]: Done.

File: src/libstore/tests/path.cc [hunk 3]
```diff
@@ -1,144 +0,0 @@
...
-TEST_DO_PARSE(lower_case, "foo")
-TEST_DO_PARSE(upper_case, "FOO")
-TEST_DO_PARSE(plus, "foo+bar")
-TEST_DO_PARSE(dash, "foo-dev")
-TEST_DO_PARSE(underscore, "foo_bar")
-TEST_DO_PARSE(period, "foo.txt")
-TEST_DO_PARSE(question_mark, "foo?why")
-TEST_DO_PARSE(equals_sign, "foo=foo")
-
-#undef TEST_DO_PARSE
```
[the remark this answers is not in the document]
Comment (roberth) [reply src/libstore/tests/path.cc:64]: Done.

**Review by Ericson2314 — APPROVED** [review]
Looks good. Thank you for taking this off my plate!

**Comment by roberth** [comment]
Yeah, working to get the release in a good state finally. I shouldn't need to do this, fwiw.

**Comment by roberth** [comment]
Finger crossed for a good release branch :confused:

**Pull request merged**